src/xtopcom/xvledger/src/xvstate.cpp

```diff
@@ -147,6 +147,18 @@ namespace top
             
             return nullptr;
         }
+
+        std::set<std::string> xvexestate_t::get_all_property_names() 
```

Comment:
need 'const'

Reply:
cannot do that because "get_mutex()" returns a non-const reference and it will be modify